qml/ConnectionMenu.qml

```diff
@@ -0,0 +1,216 @@
+import QtQuick 2.11
+import QtQuick.Controls 2.5
```

Comment:
should we be using `QtQuick.Controls 2.2`, as we have everywhere else in the project?